apps/site/api/urls.py

```diff
@@ -135,9 +135,18 @@
         r'^photos/(?P<pk>[0-9]+)/rotate-right/$',
         views.rotate_right,
         name='rotate-right'),
+    url(r'^tags/(?P<term>\w+)/$',
+        views.TagList.as_view(),
+        name='tag-list'),
     url(r'^tags/$',
         views.TagList.as_view(),
         name='tag-list'),
+    url(r'^searchbytag/(?P<tag>\w+)/$',
+        views.TagSearchList.as_view(),
+        name='tag-search-list'),
+    url(r'^searchbytag/$',
+        views.TagSearchList.as_view(),
+        name='tag-search-list'),
```

Comment:
Im not sure if this is the API interface we want so I backed out this endpoint. Currently you can search by tag for specific media using the sql endpoint.